google/cloud/speech/connection.py

```diff
@@ -26,7 +26,7 @@ class Connection(base_connection.JSONConnection):
     API_VERSION = 'v1beta1'
     """The version of the API, used in building the API call's URL."""
 
-    API_URL_TEMPLATE = '{api_base_url}/{api_version}/speech:{path}'
+    API_URL_TEMPLATE = '{api_base_url}/{api_version}/{path}'
```

Comment:
The docs for both [syncrecognize](https://cloud.google.com/speech/reference/rest/v1beta1/speech/syncrecognize) and [asyncrecognize](https://cloud.google.com/speech/reference/rest/v1beta1/speech/asyncrecognize) have the `speech:` prefix.  It looks like @daspecster made it an explicit part of the `api_request` calls below.